mods/seasonal/abilities.js

```diff
@@ -14,6 +14,23 @@ let BattleAbilities = {
 	},
 	*/
 	// Please keep abilites organized alphabetically based on staff member name!
+	// 5gen
+	seasonsgift: {
+		desc: "If the weather is sunnyday this pokemon has +1 atk and +2 spe.",
+		shortDesc: "If sunnyday is active, +1 atk & +2 spe.",
```

Comment:
Clarify this as 1.5x and 2x since its not an actual boost (I see why its done this way though)